libqtile/widget/battery.py

```diff
@@ -345,6 +346,7 @@ def __init__(self, **config) -> None:
 
         self._battery = self._load_battery(**config)
         self._has_notified = False
+        self.timeout = int(self.notification_timeout * 1000)
```

Comment:
Maybe we can do away with this line and accept a millisecond argument

Reply:
Oh nevermind, I merged :laughing:

Reply:
I thought about that. Kept it as seconds as that's consistent with `update_interval` for other widgets.